api/src/main/java/net/kyori/adventure/text/JoinConfiguration.java

```diff
@@ -118,6 +118,19 @@ public interface JoinConfiguration extends Buildable<JoinConfiguration, JoinConf
     return JoinConfigurationImpl.STANDARD_NEW_LINES;
   }
 
+  /**
+   * Provides a join configuration with no prefix or suffix that simply joins the components together using the {@link Component#space()} component.
+   *
+   * <p>A purely text based example of this syntax, without introducing the concepts of components, would join the two strings 'hello' and 'there' together,
+   * creating the following output: 'hello there'.</p>
+   *
+   * @return the join configuration
+   * @since 4.14.0
```

Comment:
```suggestion
   * @since 4.15.0
```

4.14.0 has already been released